lib/blockscore/errors/api_error.rb

```diff
@@ -24,8 +24,8 @@ def initialize(response)
     end
 
     def to_s
-      status_string = @http_status ? "(Status: #{@http_status})" : ''
-      type_string = @error_type ? "(Type: #{@error_type})" : ''
+      status_string = @http_status ? "(Status: #{@http_status})" : ""
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.